app/auth/components/Login.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Container, Grid, Form, Button, Card } from 'semantic-ui-react';
+
+const styles = {
+  root: {
+    marginTop: '7em',
+  },
+  formLabels: {
+    textAlign: 'left',
+  },
+};
+
+const Login = () => (
+  <Container style={styles.root}>
```

Comment:
Now that we are using the margin-top everywhere, I don't see any point in doing inline styling. Just  use `main` from `globals.scss`

Reply:
I think we should keep a separate CSS file for each component, like `Login.scss` for `Login.js`, it would be better than using `globals.scss` for all components.

Reply:
Yeah but `main` is in almost all views. So that qualifies as a `global` configuration. I am not telling you to put everything in `globals.scss`, just this one, because it's in almost every page. We can put different `.scss` for the configs which are typical to that view.